src/ansible/roles/ad/tasks/main.yml

```diff
@@ -1,3 +1,17 @@
+- name: 'Create netbios name for a machine in cloud'
+  set_fact:
+    netbios: "AD{{ lookup('password', '/tmp/ad_netbios chars=ascii_uppercase,digits length=4', seed=ansible_host) }}"
+  when: 'ansible_host != "172.16.200.10"'
```

Comment:
Relying on IP address is not very nice. Wouldn't it be better to introduce a variable like `CONTAINERS=yes/no` or `TARGET_ENVIRONMENT=containers|idm-ci|...possible-future-envs`?

Reply:
Also instead of settings facts conditionally here (and possibly other places), wouldn't be better to put it to idm-ci specific variable file?

Reply:
This specific address is hardcoded all around this playbook so relaying on it does not seem so bad.
Having variable and variable file for a specific environments sounds like a good idea.

As for the this AD playbook: I was hoping that it would be easy to make it work so I can get the framework running properly in idmci on VMs, and enable others to work on this. The end goal is to use the idm-ci playbooks that have support for ad forests instead.

Reply:
AD playbook is not used after all.